python-regression/tests/features/machine4/4_api_tests.feature

```diff
@@ -173,7 +173,6 @@ Feature: Test API calls on Machine 1
 		Given "getBalances" is called on "nodeA-m4" with:
 		|keys           |values			        |type               |
 		|addresses      |TEST_EMPTY_ADDRESS	        |staticList         |
-		|threshold      |100			        |int                |
```

Comment:
this should have been part of #1768 
usually I would say to move this to another PR but I am getting soft